packages/mui-joy/src/CircularProgress/CircularProgress.tsx

```diff
@@ -128,6 +128,14 @@ const CircularProgressRoot = styled('span', {
         ...rest,
       },
     }),
+    ...(ownerState.variant === 'soft' && {
+      '--CircularProgress-trackColor': 'var(--joy-palette-neutral-softBg)',
+      '--CircularProgress-progressColor': 'var(--joy-palette-primary-solidBg)',
```

Comment:
@siriwatknp how can I use something like: 
```ts
'--CircularProgress-progressColor': `var(--joy-palette-${color}-solidBg)`,
``` 

It doesn't work this way, and the way I did it, the other colors aren't going to work.

Reply:
Does this work?

`'--CircularProgress-progressColor': `var(--joy-palette-${ownerState.color}-solidBg)`,`

Reply:
Does this work?

`'--CircularProgress-progressColor': `var(--joy-palette-${ownerState.color}-solidBg)`,`

Reply:
Yep, it works perfectly! Thanks @siriwatknp 🙏